src/test/groovy/org/prebid/server/functional/model/request/auction/DoohExtData.groovy

```diff
@@ -0,0 +1,19 @@
+package org.prebid.server.functional.model.request.auction
+
+import groovy.transform.EqualsAndHashCode
+import groovy.transform.ToString
+import org.prebid.server.functional.util.PBSUtils
+
+@ToString(includeNames = true, ignoreNulls = true)
+@EqualsAndHashCode
```

Comment:
Same here

Reply:
Without it, test case: `PBS shouldn't remove bidRequest.dooh when PBS support ortb 2.6` will drop